app/src/main/java/com/movies/presentation/base/adapter/BaseAdapter.kt

```diff
@@ -15,10 +15,13 @@ abstract class BaseAdapter<MODEL : Any> :
 
     abstract class BaseViewHolder<MODEL : Any>(binding: ViewBinding) :
         RecyclerView.ViewHolder(binding.root) {
-        abstract fun onBind(item: MODEL, onClickCallback: ((MODEL) -> Unit)?)
+        open fun onBind(item: MODEL, onClickCallback: ((MODEL) -> Unit)?){
+
+        }
```

Comment:
you can fetch this to 1 line ->  open fun onBind(item: MODEL, onClickCallback: ((MODEL) -> Unit)?){ }

Reply:
done